src/coreclr/src/tools/ILVerify/README.md

```diff
@@ -8,11 +8,11 @@ The main users of this tool are people working on software that emits MSIL code.
 
 ## How to use ILVerify
 
-ILVerify is published as a global tool in the .NET runtime nightly feed. Install it by running:
+ILVerify is published as a global tool. Install it by running:
 
 ```
 dotnet new tool-manifest
-dotnet tool install dotnet-ilverify --version 5.0.0-preview* --add-source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json
+dotnet tool install dotnet-ilverify
```

Comment:
`tool-manifest` line is not needed on an average machine with `dotnet` in PATH.

Maybe we can just replace this block with reference to: https://www.nuget.org/packages/dotnet-ilverify/ that has formal instructions (and the web view get adapted for tools-like and template-like .NET packages):

<img width="769" alt="image" src="https://user-images.githubusercontent.com/3840695/100011421-5e1cd380-2dda-11eb-8c97-73a203f34b47.png">
<img width="763" alt="image" src="https://user-images.githubusercontent.com/3840695/100011815-f4e99000-2dda-11eb-9d09-e13ed1edaa45.png">
etc.

Reply:
The tool-manifest is needed unless you are installing the tool as global. I do not think it matters whether dotnet is on the PATH or not.

I have simplified the instructions to suggest installing it as global tool.

Reply:
Ah true, I didn't realized that it's global vs. local. There is a third (infamous) scenario, when dotnet is not in PATH (self-extracted runtime archive), and to run any (global) tool, we need to first DOTNET_ROOT environment variable (was doing that in containers).